aws/cloudformation/metaflow-cfn-template.yml

```diff
@@ -64,6 +64,30 @@ Parameters:
     Default: 'aws'
     AllowedValues: ['aws', 'aws-us-gov']
     Description: 'IAM Partition (Select aws-us-gov for AWS GovCloud, otherwise leave as is)'
+  ExistingVpcId:
+    Type: String
+    Default: ''
```

Comment:
Spaces are generally acceptable as empty string parameters, and I admittedly like the visual appeal of that here. 
However, I'm seeing a bit of an odd behavior with Service Catalog getting grumpy about missing "required" parameters.  It's likely to be an issue on the AWS side, but I might suggest using a different placeholder for these just to cover that use-case.